src/stan-users-guide/proportionality-constants.Rmd

```diff
@@ -0,0 +1,150 @@
+# Proportionality Constants  {#proportionality-constants.chapter}
+
+When evaluating a likelihood or prior as part of the log density computation in
+MCMC, variational inference, or optimization, it is usually only necessary to
+compute the functions up to a proportionality constant (or similarly compute
+log densities up to an additive constant). In MCMC this comes from the fact that
+the distribution being sampled does not need to be normalized (similarly
```

Comment:
Also in relation to your question about terminology, maybe just make the connection between "normalized" and the "constants" that you previously mentioned explicit here. Then I think you can freely use all the terms below and not worry about it.